streampark-console/streampark-console-service/src/test/java/org/apache/streampark/console/core/service/ApplicationServiceTest.java

```diff
@@ -89,4 +94,15 @@ public void start() throws Exception {
         applicationService.start(application, false);
     }
 
+    @Test
+    public void uploadTest() throws Exception {
```

Comment:
I think we need to add another test for verifying the uploaded file is not a jar file.
It looks like we just verify the ContentType

Reply:
> I think we need to add another test for verifying the uploaded file is not a jar file. It looks like we just verify the ContentType

Thanks very much for your review, next we will provide test cases to verify the upload file type.